hosting/src/app/games/quiz/lobby/lobby.component.html

```diff
@@ -70,7 +70,7 @@ <h2>Quiz Overview</h2>
                     <brand-expansion label="{{ round.title }} ({{ round.questionList | count }})" (onToggle)="loadQuestions(round)">
                         <ng-container *ngIf="round.questions; else noQuestions">
                             <div class="question" *ngFor="let question of round.questions">
-                                <div>{{ question.text | slice: 0 : 30 }}</div>
+                                <div><span class="question-number"></span> {{ question.text | slice: 0 : 29 }}</div>
```

Comment:
the span to provide a class isn't really needed, you can just use the div in the css selector

```
.question {
    > div {
        &::before {
            ......
        }
    }
}
```

keeps the HTML a bit cleaner (and I don't like empty html tags :)

Reply:
Done.
We could even get rid of the inner div by adding adding display:flex to the question class like this:
` .question {
    ...
        display:flex;
        &::before {
            color: $brand-primary;
            ...
        }
    }`

I guess you put it there to give it further use in the future so I kept it.